test/hotspot/jtreg/compiler/intrinsics/bmi/TestBzhiI2L.java

```diff
@@ -37,11 +37,16 @@
 
 package compiler.intrinsics.bmi;
 
+import jdk.test.lib.Platform;
 import sun.hotspot.cpuinfo.CPUInfo;
 
 public class TestBzhiI2L {
 
     public static void main(String args[]) throws Throwable {
+        if (Platform.isX86()) {
+            System.out.println("INFO: Bzhiq not implimented for x86_32, test SKIPPED" );
```

Comment:
typo: `implimented`
should be `implemented`

Reply:
Agreed. Thanks Vladimir.